front/ticket_user.form.php

```diff
@@ -48,6 +48,15 @@
 if (isset($_POST["update"])) {
    $link->check($_POST["id"], UPDATE);
 
+   if (isset($_POST['alternative_email']) && !NotificationMail::isUserAddressValid($_POST['alternative_email'])) {
```

Comment:
I'm not sure about this part, i think it would be better to move it in Ticket_User::prepareInputForUpdate.